City-of-Pori/pori-kada
# Pori 77 additional information element

To test:

1. drush updb -y && drush fra -y && drush cc all
2. Go to edit some page and make sure there is the "Additional information" tab. Add there some internal and external links and attachments.
3. To add image as an attachment you first need to add the image as a FILE not an IMAGE into scald. After you have done this, you can drag & drop it to the "File attachments" field.
4. Also make sure that the subpages are no longer displayed on the page and have been moved behind the "Unused" tab.
